Selects

Kwolik opened the pull request:
Repair all selects and footer in adding work

Kwolik committed:
repair Select

Kwolik committed:
repair Select in PersonalInformation

Kwolik committed:
Add marginTop

Kwolik committed:
repair Select and footer

Kwolik committed:
value=""

upsu90 commented on web-client/src/components/PersonalInformation.jsx:
```diff
@@ -242,60 +245,48 @@ export default function PersonalInformation(props) {
+              inputRef={register}
+              required
+              id="academic-title-signup"
               name="specialization"
-              className={style.formControl}
+              label="Specialization"
+              autoComplete="job-title"
```
it isn't an academic title or job title so please change id and autoComplete

upsu90 commented on web-client/src/components/PersonalInformation.jsx:
```diff
@@ -242,60 +245,48 @@ export default function PersonalInformation(props) {
+              <MenuItem className={style.MenuItem} value={"Chemistry"}>
+                Chemistry               
+              </MenuItem>
+              <MenuItem className={style.MenuItem} value={"Psychics"}>
+                Psychics               
```
please change to Physics

upsu90 commented on web-client/src/components/SignUpForm.jsx:
```diff
@@ -289,57 +292,46 @@ export default function SignUpForm(props) {
+              <MenuItem className={style.MenuItem} value={"Chemistry"}>
+                Chemistry               
+              </MenuItem>
+              <MenuItem className={style.MenuItem} value={"Psychics"}>
+                Psychics               
```
Physics :p

upsu90 commented on web-client/src/components/SignUpForm.jsx:
```diff
@@ -289,57 +292,46 @@ export default function SignUpForm(props) {
+              inputRef={register}
+              required
+              id="academic-title-signup"
               name="specialization"
-              className={style.formControl}
+              label="Specialization"
+              autoComplete="job-title"
```
id and autoComplete again

upsu90 commented on web-client/src/pages/adding-work.js:
```diff
@@ -141,6 +142,10 @@ function AddingWork() {
       .max(maxAuthors, "You cannot add more authors"),
+    specialization: yup.string().when("specializations", {
+      is: (specializations) => specialization === "Select",
+      then: yup.string().required("Time field is required"),
```
why "Time field"?

upsu90 commented on web-client/src/pages/adding-work.js:
```diff
@@ -357,58 +357,46 @@ function AddingWork() {
+              <MenuItem className={style.MenuItem} value={"Chemistry"}>
+                Chemistry               
+              </MenuItem>
+              <MenuItem className={style.MenuItem} value={"Psychics"}>
+                Psychics               
```
Physics

upsu90 reviewed (changes requested):
placeholder should contain "Select"

Kwolik committed:
Fixed my mistake

KamilDonda reviewed (commented):
+ ![image](https://user-images.githubusercontent.com/44376350/104292853-9fb62a80-54bd-11eb-8c33-a6ec8823a5d5.png)

+ when I choose "Select" I don't get an error: 
![image](https://user-images.githubusercontent.com/44376350/104294927-1eac6280-54c0-11eb-8d6d-3b2ac8136f14.png)



//EDIT:

`value={""}` -> `value={"Select"}` fixes this error

Kwolik committed:
Fixed validation

Kwolik committed:
Merge branch 'master' into selects

# Conflicts:
#	web-client/src/pages/adding-work.js

Kwolik committed:
Create one elements in MenuItem

Kwolik committed:
Mapping in selects

# Conflicts:
#	web-client/src/components/PersonalInformation.jsx
#	web-client/src/components/SignUpForm.jsx
#	web-client/src/pages/adding-work.js

Kwolik committed:
Merge branch 'master' into selects

# Conflicts:
#	web-client/src/pages/adding-work.js
#	web-client/src/pages/work-view.js

Kwolik committed:
Change one element in categories -> value
